docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+This tutorial provides a step-by-step guide on how to run BigDL-Orca programs on Kubernetes (K8s) clusters, using a [PyTorch Fashin-MNIST program](https://github.com/intel-analytics/BigDL/tree/main/docs/docs/tutorials/tutorial_example/Fashion_MNIST/) as a working example.
+
+# 1. Key Concepts
+## 1.1 Init_orca_context
+A BigDL Orca program usually starts with the initialization of OrcaContext. For every BigDL Orca program, you should call `init_orca_context` at the beginning of the program as below:
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode, master, container_image, 
+                  cores, memory, num_nodes, driver_cores, driver_memory, 
+                  extra_python_lib, penv_archive, jars, conf)
+```
+
+In `init_orca_context`, you may specify necessary runtime configurations for running the example on K8s, including:
+* `cluster_mode`: a String that specifies the underlying cluster; valid value includes `"local"`, `"yarn-client"`, `"yarn-cluster"`, `"k8s-client"`, `"k8s-cluster"`, `"bigdl-submit"`, `"spark-submit"`, etc.
+* `master`: a URL format to specify the master address of K8s cluster.
+* `container_image`: a String that specifies the name of docker container image for executors.
+* `cores`: an Integer that specifies the number of cores for each executor (default to be `2`).
+* `memory`: a String that specifies the memory for each executor (default to be `"2g"`).
+* `num_nodes`: an Integer that specifies the number of executors (default to be `1`).
+* `driver_cores`: an Integer that specifies the number of cores for the driver node (default to be `4`).
+* `driver_memory`: a String that specifies the memory for the driver node (default to be `"1g"`).
+* `extra_python_lib`: a String that specifies the path to extra Python packages, one of `.py`, `.zip` or `.egg` files (default to be `None`).
+* `penv_archive`: a String that specifies the path to a packed Conda archive (default to be `None`).
+* `jars`: a String that specifies the path to needed jars files (default to be `None`).
+* `conf`: a Key-Value format to append extra conf for Spark (default to be `None`).
+
+__Note__: 
+* All arguments __except__ `cluster_mode` will be ignored when using `spark-submit` to submit and run Orca programs.
```

Comment:
missing bigdl-submit?

Reply:
> missing bigdl-submit?

bigdl-submit is not used on k8s, do we need to mention it?